glide.yaml

```diff
@@ -264,3 +264,9 @@ import:
 - package: gonum.org/v1/gonum
   repo: https://github.com/gonum/gonum.git
   version: cebdade430ccb61c1feba4878085f6cf8cb3320e
+
+# set alias location for this package due to
+# https://github.com/Masterminds/glide/issues/1057; upstream did this in
+# https://github.com/kubernetes/kubernetes/pull/72138
+- package: bitbucket.org/ww/goautoneg
+  repo: https://github.com/munnerz/goautoneg.git
```

Comment:
is that what upstream did? 

this does the trick without changing the repo location https://github.com/openshift/library-go/pull/469/files#diff-395f41b2a27b70ce44399c91c82158c2R68

Reply:
We don't run glide in upstream (went straight from Godeps to go modules) -- we just straight up replaced the dependency with the forked copy. We were also seeing flakiness in bitbucket for CI (but obviously at a much higher PR rate).

We already import the forked dependency too, because of the upstream Kubernetes changes. I think either change is acceptable with a slight preference to move to the github fork to align with upstream.

Let me know if you'd like me to update my PR, @tnozicka :)

Reply:
@tnozicka I confirmed that locally, this fixes openshift/origin for me as well. Happy to go with either option.

Reply:
given upstream switched to that fork, bot options look ok to me

Reply:
/unassign
/assign @deads2k